conf/app.ini

```diff
@@ -203,7 +203,8 @@ IMPORT_LOCAL_PATHS = false
 ; Whether to allow signin in via OpenID
 ENABLE_OPENID_SIGNIN = true
 ; Whether to allow registering via OpenID
-ENABLE_OPENID_SIGNUP = true
+; Do not include to rely on DISABLE_REGISTRATION setting
+;ENABLE_OPENID_SIGNUP = true
```

Comment:
Migrating variablenames could be done by removing the old default from `conf/app.ini` but load it from `custom/conf/app.ini` if it exists. then `cfg = oldVar; if exists(newVar) { cfg = newVar }` in `LoadConfig`. Maybe throw a deprication warning. So no migration needed. And would make the config defautls easier. Since `ENABLE_OPENID_SIGNUP = ! DISABLE_REGISTRATION` which I don't think you can do in `ini` format 😒